scripts/dotnet.py

```diff
@@ -9,13 +9,15 @@
 from glob import iglob
 from json import loads
 from logging import getLogger
-from os import chmod, environ, makedirs, path, pathsep
+from os import chmod, environ, listdir, makedirs, path, pathsep
 from stat import S_IRWXU
 from subprocess import check_output
 from sys import argv, platform
 from urllib.parse import urlparse
 from urllib.request import urlopen, urlretrieve
 
+import re
```

Comment:
not related to the PR but to the language: python named the regex module just `re`? that's crazy

Reply:
Yes, agreed :( … how hard it would have been to name it something like: `regex`, `regular_expression`